src/cmds/scripts/pbs_server

```diff
@@ -35,8 +35,17 @@
 # "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
 # trademark licensing policies.
 
+# save env variables in a temp file
+env_save="/tmp/$$_`date +'%s'`_env_save"
```

Comment:
Optional comment: use `$(date +'%s')` instead `` `date +'%s'` ``

Reply:
Sure, but since this code is identical to the code in the init script, I'll have to change the init script as well.  Looking at other scripts in src/cmds/scripts, there are others that will suffer from the same issue.  Maybe I should file a ticket for them.